app/frontend/src/submissions/views/SubmissionsHome.vue

```diff
@@ -574,14 +576,121 @@ export default {
 
         // non-form fields that should be saved with form
         meta: {
-          drillerSameAsPersonResponsible: false
+          drillerSameAsPersonResponsible: false 
```

Comment:
```suggestion
          drillerSameAsPersonResponsible: false
```
Sneaky sneaky white space